interfaces/sourcegen/src/sourcegen/csharp/config.yaml

```diff
@@ -16,18 +16,23 @@ ignore_funcs:
   ctfunc_auto.yaml:
     - new_advanced
 
-# Return type crosswalks
-ret_type_crosswalk:
+# C type crosswalks;
+# map C type names to C# type names where they differ
+c_type_crosswalk:
   const char*: string
   const double*: ReadOnlySpan<double>
-  const int*: ReadOnlySpan<int>
+  const int32_t*: ReadOnlySpan<int>
   char*: Span<byte>
   double*: Span<double>
+  int32_t*: Span<int>
+  int32_t: int
 
-# Parameter type crosswalks
-prop_type_crosswalk:
+# Span type crosswalks;
+# map Span type used in interop call to appropriate return type
+span_type_crosswalk:
```

Comment:
Would you be OK with keeping the original name (`prop_type_crosswalk`) or maybe `span_prop_type_crosswalk`. This crosswalk is exclusive to the bit where we create the .NET properties for the higher-level classes. For any other stuff where we would hand-build anything, my thought is to stick to the low-level stuff being added to the `LibCantera` class.

Reply:
Absolutely! Let's work to add docstrings (XML API documentation comments, in full, in the .NET world), params, heritage, all of it! My vision is that all user access to Cantera from C# will be through the high-level classes, so it will need excellent documentation. I think we will be able to directly scaffold roughly half of the high-level classes, and write the other half in a way that is idiomatic to a .NET developer (see my `SpeciesCollection` as an example).